OutOfSchool/OutOfSchool.WebApi/Models/ChildDTO.cs

```diff
@@ -36,7 +37,7 @@ public class ChildDto
 
         public Guid ParentId { get; set; } = default;
 
-        public long? SocialGroupId { get; set; } = default;
+        public List<SocialGroupDto> SocialGroups { get; set; } = new List<SocialGroupDto>();
```

Comment:
Such initializers telling me that this model can have different states when some of fields can be uninitialized and causes NRE in some parts of code.
Maybe we have to have more strict checks in the code to avoid such  initializers ?